benchmarking/README.md

```diff
@@ -4,6 +4,9 @@ This is work in progress
 
```

Comment:
I am not very happy with the squashing of commits, they are no longer containing logical changes but it looks like a too big task to go back. Anyway, minimum you need to do is to sanitize the commit messages of the squashed commits, for example the `test/statistics-analysis: Save NRI resource policy and runtime logs` needs to be fixed.

Reply:
I changed the commit history by a lot. Now the commits add separate pieces of functionality.